### PR TITLE
 now total 26 categories

### PR DESCRIPTION
**Pull request by AndyHoang**
Just added some catagories, tested with linux build.

Again, thanks for your support linux community.

https://github.com/viktorgullmark/exilence-next/issues/709

### CODE REVIEW

File: ExilenceNextApp/src/services/poe-ninja.service.ts [hunk 1]
```diff
@@ -50,6 +50,12 @@ function getItemCategories() {
     'Beast',
     'Vial',
     'Invitation',
+    'Artifact',
+    'UniqueJewel',
+    //'ClusterJewel',
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/services/poe-ninja.service.ts:55]: Reason for commenting this out?

File: ExilenceNextApp/src/services/poe-ninja.service.ts [hunk 2]
```diff
@@ -50,6 +50,12 @@ function getItemCategories() {
...
+    'BlightedMap',
+    //'BaseType',
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/services/poe-ninja.service.ts:57]: Reason for commenting this out?

Comment (AndyHoang) [reply ExilenceNextApp/src/services/poe-ninja.service.ts:57]: I added it in commit msg. So basically those are usually medium confidence and sometimes misleading, so I decided to comment it out. I can add it back if you think its make sense.

File: ExilenceNextApp/src/services/poe-ninja.service.ts (hunk 1 above)
Comment (AndyHoang) [reply ExilenceNextApp/src/services/poe-ninja.service.ts:55]: I added it in commit msg. So basically those are usually medium confidence and sometimes misleading, so I decided to comment it out. I can add it back if you think its make sense.

Comment (viktorgullmark) [reply ExilenceNextApp/src/services/poe-ninja.service.ts:55]: Alright, I think we should keep them for now at least, then decide what we do with them for the next league

Comment (AndyHoang) [reply ExilenceNextApp/src/services/poe-ninja.service.ts]: updated

**Comment by viktorgullmark** [comment]
Sorry to bug you again, I was mistaken.. I thought you removed these, didnt realize they werent in our code in the first place. 😨 

I totally agree with you, if we dont have appropriate code in place to identify these with high confidence; leave them out for now and only add the ones that seem to be accurately priced!

**Review by sbsrnt — APPROVED** [review]

**Comment by AndyHoang** [comment]
No worries, to make sure that we dont have any further miscommunication, I will:
- commented out those 2 item: BaseType and cluster jewels 
This because those mostly not accuracy pricing. Maybe late we can fllter out those "low-medium" confidence items, to improve net worth estimation.

**Comment by sbsrnt** [comment]
I would also comment out Helmet Enchants as these are not so easily priced. What makes a good price for the helmet enchant is on which helmet we apply the enchant into.

**Comment by sbsrnt** [comment]
Actually UniqueJewels may also be an issue if we take watcher's eyes into consideration. Similarly voices from Delirium as we probably won't be able to distinguish 1-3-5-7 passives.

**Comment by AndyHoang** [comment]
yep, let be safer than sorry especially about "money" ;)

**Comment by viktorgullmark** [comment]
Great work, and sorry for the delay, took a pause from the project

**Pull request merged**